### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by xxUnknown13xx** [trclassic92/tr-dumpsterdive#1]
Is there a way to take off that mini game? I would like to only let them just search without having them play a mini game. Great script btw :)

**Comment by trclassic92** [comment]
Just remove it

**Comment by xxUnknown13xx** [comment]
I tried to install this, but it didn't work for me..

**Comment by trclassic92** [comment]
What do you mean it doesn't work for you? 

This new statement just doesn't make sense to what you said before. You asked if this script was able to work with out the minigame which it is able to. All you do is remove the code that supports the mini-game and boom done

**Comment by xxUnknown13xx** [comment]
Yes I had asked about the mini game before I tried to install it. Once I installed it, none of the dumpsters worked. I can't seem to search them :/ 
I added the folders in the right place. Did the "start" start and did "insure".

**Comment by poolshark120502** [comment]
![image](https://github.com/trclassic92/tr-dumpsterdive/assets/72841575/ac9c6819-6547-475f-ba37-6c945a83a198)
anyway turn this off floods my f8

**Comment by trclassic92** [comment]
This is a print error coming from ox_target so you can look for that in the lua file or somewhere in the json ( but I highly doubt that its in the json file)
